plugins/inputs/dns/README.md

```diff
@@ -0,0 +1,51 @@
+# DNS Query Input Plugin
+
+The DNS plugin gathers dns query times in miliseconds - like [Dig](https://en.wikipedia.org/wiki/Dig_\(command\))
+
+### Configuration:
+
+```
+# Sample Config:
+[[inputs.dns_query]]
+  ### Domains or subdomains to query
+  domains = ["mjasion.pl"] # required
```

Comment:
Isn't there a way to do this without `domains` being required? I just ask because with `dig` you can run `dig 8.8.8.8 any` without specifying a domain and get your query time, with domains being optional arguments.